.github/workflows/review.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           fi
       - name: Run eslint
         if: env.run_eslint == 1
-        uses: reviewdog/action-eslint@v1
+        uses: reviewdog/action-eslint@v1.12.1
```

Comment:
v1 means pull latest v1+, doesn't it?

Reply:
Yeah, it's the wrong change (and it didn't work either). I'll drop this for each of the PRs that have it (and I'll probably drop the zstd fix too).